site/docs/ch/optimizing.md

```diff
@@ -1,39 +1,97 @@
-# Optimizers
+# Optimizing
```

Comment:
shall we split optimizing into two chapters like optimizing type and optimizers?